controller/persistence/scratchpad.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Seems like a test script. Maybe better placed in the utils folder located in the root of the repo?